src/graphics/device.js

```diff
@@ -1138,17 +1138,28 @@ Object.assign(pc, function () {
                 this.renderTarget = prevRt;
                 gl.bindFramebuffer(gl.FRAMEBUFFER, prevRt ? prevRt._glFrameBuffer : null);
             } else {
-                if (!this._copyShader) {
-                    var chunks = pc.shaderChunks;
-                    this._copyShader = chunks.createShaderFromCode(this, chunks.fullscreenQuadVS, chunks.outputTex2DPS, "outputTex2D");
-                }
+                var shader = this.getCopyShader();
                 this.constantTexSource.setValue(source._colorBuffer);
-                pc.drawQuadWithShader(this, dest, this._copyShader);
+                pc.drawQuadWithShader(this, dest, shader);
             }
 
             return true;
         },
 
+        /**
+         * @function
+         * @name pc.GraphicsDevice#createFrameBuffer
```

Comment:
createFrameBuffer -> getCopyShader?

Reply:
And can we leave this function as private for now?

Reply:
Sorry, fixed the name and made it private.